public/css/style.scss

```diff
@@ -0,0 +1,137 @@
+@charset "utf-8";
+/* CSS Document */
+
+/* ---------- FONTAWESOME ---------- */
+/* ---------- http://fortawesome.github.com/Font-Awesome/ ---------- */
+/* ---------- http://weloveiconfonts.com/ ---------- */
+
+@import url(http://weloveiconfonts.com/api/?family=fontawesome);
```

Comment:
Copying css is not a good idea if you don't know why you are using it! Please make sure you check if things are working cross browser!